package/yast2-services-manager.changes

```diff
@@ -7,11 +7,6 @@ Wed Jul 25 12:17:42 UTC 2018 - jlopez@suse.com
   on demand or manually) (fate#319427).
 - 4.0.4
 
--------------------------------------------------------------------
-Thu Jun 28 16:45:33 UTC 2018 - jlopez@suse.com
-
-- Added button to show journal logs (related to fate#319427).
```

Comment:
It is OK to drop this line because this PR has not been merged yet.

Reply:
Yes, we are safe :)